customcontext/texture/eglgralloctexture.cpp

```diff
@@ -318,7 +318,7 @@ void EglGrallocTexture::bind()
 
         int error;
         while ((error = glGetError()) != GL_NO_ERROR)
-            qDebug() << "Error after glEGLImageTargetTexture2DOES" << hex << error;
+            qDebug() << "Error after glEGLImageTargetTexture2DOES" << Qt::hex << error;
```

Comment:
Qt::hex introduced in 5.14 so this needs some backward support.

Reply:
Fixed with https://github.com/sailfishos/qtscenegraph-adaptation/pull/1/commits/bb1d0094d4710788fe3c78f6a103f7d237c04967